week-1/errors/4.js

```diff
@@ -1,2 +1,4 @@
 const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+const 24hourClockTime = "08:53";
+//variable names in JavaScript cannot start with a digit. They must start with a letter, underscore (_), or dollar sign ($).
```

Comment:
You've identified the problem correctly. Don't forget to fix the statements to fix the error